modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner (HPP) custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, you are not required to change your HPP custom resource. You continue using your existing storage classes. You do not have to modify the HPP custom resource for the legacy provisioner DaemonSet.
+
+== Creating the legacy HPP custom resource
+
+If you are installing {VirtProductName} version 4.10 on a new cluster, you must create an HPP legacy custom resource:
+
+[source,yaml]
+----
+apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+kind: HostPathProvisioner
+metadata:
+  name: hostpath-provisioner
+spec:
+  imagePullPolicy: IfNotPresent
+  pathConfig:
+    path: "/var/hpvolumes"
+    useNamingPrefix: false
+ workload:
+   nodeSelector:
+     kubernetes.io/os: linux
+----
+
+== Creating the CSI driver provisioner custom resource
+
+The legacy provisioner DaemonSet and the CSI driver DaemonSet are supported in parallel for a number of releases. However, at some point, the legacy provisioner DaemonSet will no longer be supported. To prepare for this change,
+create a CSI driver custom resource using a basic storage pool.
```

Comment:
It sounds like this section should just be deleted.

- Is there value in listing the contents of the legacy HPP CR anywhere in this doc?
- Should we state that at some point support for HPP legacy CR will go away? Or leave that deprecation for the release notes. In the notes, there is a specific section for deprecated features and this should probably be included starting in 4.10.

Reply:
I agree with Alexander, let's remove the DaemonSet part. 
- HPP legacy CR - doesn't seem worth listing its contents in this doc. 
- Deprecation - can be mentioned in release notes. I don't have a strong opinion if we need to mention it here :)

Reply:
`To prepare for this change, create a CSI driver custom resource using a basic storage pool.` is kind of confusing to me. If you have a legacy hpp CR, it will create BOTH the legacy provisioner AND the CSI driver for you. If you create a new cluster, you should just create the storage pool format HPP CR, which will only create the csi driver (since there are no legacy volumes to worry about). You have to do either the legacy format, or the storage pool format. You CANNOT do both. The way this reads right now it seems you can do both which is not true.

It should be something like `to prepare for this change if you have the legacy HPP CR, you can also create storage class that uses the csi provisioner (which you nicely show below). It might make sense to show the format of that storage class (it doesn't take the storagePool parameter)

Reply:
Done.